### PR TITLE
[V20 Alpha2] bump version code and addon api for v20 alpha2

### PR DESCRIPTION
**Pull request by fuzzard**
## Description
V20 Alpha 2

## Motivation and context
V20 Alpha 2

## What is the effect on users?
Release

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: tools/Linux/kodi.metainfo.xml.in [hunk 1]
```diff
@@ -78,6 +78,7 @@
         </screenshot>
     </screenshots>
     <releases>
+        <release date="2022-06-09" version="20.0a2-Nexus" type="development"><url>https://kodi.tv/article/kodi-nexus-alpha-2/</url></release>
```
Comment (HiassofT) [inline tools/Linux/kodi.metainfo.xml.in:81]: shouldn't this be 2022-07-09?

**Comment by MediaBrasil** [comment]
@fuzzard 
Please fix the line 80, correct is  </keywords>

**Comment by fuzzard** [comment]
you are going to have to give me specifics, ive only put in what was provided by someone else.

**Comment by fuzzard** [comment]
Well thats completely unrelated. If the file is incorrect, someone will need to PR the correct approach. Wont be changing in the alpha 2 release

**Comment by razzeee** [comment]
https://github.com/xbmc/xbmc/pull/21661

**Comment by Gujal00** [comment]
The Android and Windows version come bundled with different versions of Python causing difficulty in addon development.
Currently Nexus Alpha 2 on Windows comes with Python 3.8.5 and on Android comes with 3.9.13
Any chance to bump the bundled Python version to be the same on both platforms in Alpha 3? Thanks

**Comment by fuzzard** [comment]
Not likely. It's never expected to be the same on all platforms, you'll need to manage that case in your add-on

**Comment by Gujal00** [comment]
I understand on Linux and Mac it will use the system version, however Windows and Android versions come with bundled python as part of the kodi installer and they could be easily set up to be the same 3.9.13 instead of the quite old 3.8.5 on Windows. Thanks

**Comment by rainman74** [comment]
@fuzzard What do you think about not working on v21 immediately after the release of v20, but to first see v20 as a stable code base for at least the next 5 years and to develop it further in smaller steps and to release further build releases in much shorter cycles. For example v20.0.1, v20.0.2, v20.0.3, etc.

Apple does it this way with macOS, Microsoft does it this way, and I think this process has proven itself from my point of view.

It would be good for Kodi to focus on a stable basis now.

**Pull request merged**